### PR TITLE
ZCS-11425 Configure LDAP attributes for storing S3 configurations

### PR DESCRIPTION
**Pull request by inpati**
**Problem Statement**
Add LDAP attributes to store S3 configurations

**Solution**
Added New Attributes:
1. globalExternalStoreConfig -> global level
2. serverExternalStoreConfig -> server level

### CODE REVIEW

**Review by deepakzimb — COMMENTED** [review]
This PR should contain other files as well that creates after getter/setter commands

File: store/conf/attrs/zimbra-attrs.xml [hunk 1]
```diff
@@ -9929,4 +9929,10 @@ TODO: delete them permanently from here
   <defaultCOSValue>30</defaultCOSValue>
   <desc>Default calendar resolution preference</desc>
 </attr>
+<attr id="5000" name="globalExternalStoreConfig" type="string" cardinality="single" optionalIn="globalConfig" since="9.1.0">
```
Comment (dasiyogesh) [inline store/conf/attrs/zimbra-attrs.xml:9932]: there should be just one attribute which can have global as well server value
Please  refer https://github.com/Zimbra/zm-mailbox/blob/develop/store/conf/attrs/zimbra-attrs.xml#L9898

**Review by dasiyogesh — CHANGES_REQUESTED** [review]
globalExternalStoreConfig & serverExternalStoreConfig there should be just one attribute externalStoreConfig that can be set on global & server both

File: store/src/java/com/zimbra/cs/account/ZAttrServer.java [hunk 2]
```diff
@@ -224,6 +224,78 @@ public Map<String,Object> unsetDescription(Map<String,Object> attrs) {
...
+     *
+     * @param serverExternalStoreConfig new value
+     * @param attrs existing map to populate, or null to create a new map
+     * @return populated map to pass into Provisioning.modifyAttrs
+     *
+     * @since ZCS 9.1.0
+     */
+    @ZAttr(id=5001)
+    public Map<String,Object> setServerExternalStoreConfig(String serverExternalStoreConfig, Map<String,Object> attrs) {
+        if (attrs == null) attrs = new HashMap<String,Object>();
```
[the remark this answers is not in the document]
Comment (deepakzimb) [reply store/src/java/com/zimbra/cs/account/ZAttrServer.java:265]: Yes Rakesh, when we run getter/setter commands, related method generates

[the remark this answers is not in the document]
Comment (inpati) [reply store/src/java/com/zimbra/cs/account/ZAttrServer.java:265]: We can't adjust indentation in auto generated code

**Comment by inpati** [comment]
> This PR should contain other files as well that creates after getter/setter commands

Added other files as well

**Pull request closed without merging**